Dockerfile_release

```diff
@@ -8,21 +8,18 @@ COPY . /go/src/github.com/influxdata/flux
 WORKDIR /go/src/github.com/influxdata/flux
 
 # Install common packages
-RUN apt-get update && \
+RUN apt-get clean && apt-get update && \
     apt-get install --no-install-recommends -y \
         clang \
         gcc-arm-linux-gnueabihf \
         gcc-arm-linux-gnueabi \
         gcc-aarch64-linux-gnu \
         gcc-i686-linux-gnu && \
     rm -rf /var/lib/apt/lists/*
+RUN go get github.com/influxdata/pkg-config
 RUN curl https://sh.rustup.rs -sSf | \
-    sh -s -- --default-toolchain stable -y
+        sh -s -- --default-toolchain stable -y
 
-RUN cd libflux && cargo build
-RUN go build \
-        -ldflags '-extldflags "-fno-PIC -static"' \
-        -buildmode pie \
-        -tags 'libflux static_build' \
-        -o flux \
-        .
+# XXX: rockstar (13 Feb 2020) - The `-tags` portion of this command can be
+# removed when applied to feat/algo-w.
+RUN go build -tags 'libflux'
```

Comment:
Can you remove this section? This should be done as part of building by anybody running this command, but this build command shouldn't be part of the dockerfile.

Reply:
We talked about this, and I explained that until `release.sh` is updated, I needed to have some way of repeating the process.

Reply:
I'm ok if we comment out this line just to leave the instructions for how to build. Maybe add a comment saying how to use `-v` to share the directory so it can be copied and pasted.